src/sidebar/services/frame-sync.js

```diff
@@ -366,9 +366,23 @@ export class FrameSyncService {
 
     guestRPC.on(
       'showAnnotations',
-      /** @param {string[]} tags */ tags => {
-        this._store.selectAnnotations(this._store.findIDsForTags(tags));
+      /**
+       * @param {string[]} tags
+       * @param {boolean} focusFirstInSelection - Whether to give keyboard focus
+       *   to the first annotation in the selection.
+       */
+      (tags, focusFirstInSelection = false) => {
```

Comment:
We don't have any conventions around this, but after the addition of the new logic, this callback feels a bit "beefy" to be anonymous.